comp/core/workloadmeta/collectors/internal/kubeapiserver/reflector_store.go

```diff
@@ -144,13 +144,13 @@ func (r *reflectorStore) Delete(obj interface{}) error {
 	var uid types.UID
 	var entity workloadmeta.Entity
 	switch v := obj.(type) {
+	// All the supported objects need to be in this switch statement to be able
+	// to be deleted.
 	case *corev1.Pod:
 		uid = v.UID
-	case *corev1.Node:
```

Comment:
Nodes and namespaces were migrated to the new generic collector, so they're no longer needed here.